src/Microsoft.Identity.Web/AuthorityHelpers.cs

```diff
@@ -6,35 +6,35 @@
 
 namespace Microsoft.Identity.Web
 {
-    internal static class AuthorityHelpers
-    {
-        internal static string BuildAuthority(MicrosoftIdentityOptions options)
-        {
-            Uri baseUri = new Uri(options.Instance);
-            string pathBase = baseUri.PathAndQuery.TrimEnd('/');
-            var domain = options.Domain;
-            var tenantId = options.TenantId;
+	internal static class AuthorityHelpers
+	{
+		internal static string BuildAuthority(MicrosoftIdentityOptions options)
+		{
+			Uri baseUri = new Uri(options.Instance);
+			string pathBase = baseUri.PathAndQuery.TrimEnd('/');
+			var domain = options.Domain;
+			var tenantId = options.TenantId;
 
-            if (options.IsB2C)
-            {
-                var userFlow = options.DefaultUserFlow;
-                return new Uri(baseUri, new PathString($"{pathBase}/{domain}/{userFlow}/v2.0")).ToString();
-            }
-            else
-            {
-                return new Uri(baseUri, new PathString($"{pathBase}/{tenantId}/v2.0")).ToString();
-            }
-        }
+			if (options.IsB2C)
+			{
+				var userFlow = options.DefaultUserFlow;
+				return new Uri(baseUri, new PathString($"{pathBase}/{domain}/{userFlow}/v2.0")).ToString();
+			}
+			else
```

Comment:
This doesn't really matter that much, but the else keyword and block nesting isn't needed because the `if (options.IsB2C)` block had a return statement.